src/projects.h

```diff
@@ -348,35 +502,22 @@ extern struct PJ_DATUMS pj_datums[];
 extern struct PJ_PRIME_MERIDIANS pj_prime_meridians[];
 #endif
 
+
+
+
+
 #ifdef PJ_LIB__
-/* repetitive projection code */
+/* repetitive projection code (most of them eliminated now) */
```

Comment:
Elimaniting these macros is a very good idea. Could you create a ticket that describes this effort? It will be easier to keep track of that way, and there's a bigger change someone will pick it up and do the work.